From 95821cbad0ce5df59fcb591e5aab9459a388321e Mon Sep 17 00:00:00 2001
From: Sebastian Lackner <sebastian@fds-team.de>
Date: Fri, 10 Apr 2015 08:17:22 +0200
Subject: Revert "opengl32: Return a NULL pointer for functions requiring
 unsupported or disabled extensions."

This reverts commit bfd4836867d6d90eaeae6ccbc02e37678b59b8f1 and
                    5c9ddc55dcf6e64d747b1a8ab01e408d4b8d55e1 and
                    e44f8eaa0fce02032e6c3f43052821f40eaba53b.
---
 dlls/opengl32/wgl.c | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/dlls/opengl32/wgl.c b/dlls/opengl32/wgl.c
index 3b817a4..75d3796 100644
--- a/dlls/opengl32/wgl.c
+++ b/dlls/opengl32/wgl.c
@@ -877,23 +877,7 @@ PROC WINAPI wglGetProcAddress( LPCSTR name )
         void *driver_func = funcs->wgl.p_wglGetProcAddress( name );
 
         if (!is_extension_supported(ext_ret->extension))
-        {
-            unsigned int i;
-            static const struct { const char *name, *alt; } alternatives[] =
-            {
-                { "glCopyTexSubImage3DEXT", "glCopyTexSubImage3D" },  /* needed by RuneScape */
-            };
-
             WARN("Extension %s required for %s not supported\n", ext_ret->extension, name);
-            driver_func = NULL;
-
-            for (i = 0; i < sizeof(alternatives)/sizeof(alternatives[0]) && !driver_func; i++)
-            {
-                if (strcmp( name, alternatives[i].name )) continue;
-                WARN("Trying alternative %s for %s\n", alternatives[i].alt, name );
-                driver_func = wglGetProcAddress( alternatives[i].alt );
-            }
-        }
 
         if (driver_func == NULL)
         {
-- 
2.6.2

